### DISCUSSION TITLE
Modern project with REST API

### DISCUSSION THREAD

**Discussion by marcoschwepp** [community/community#63115] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi there,

so I'm trying to move closed issues with `state_reason = not_planned` to a specific column in my project board. 
Using existing actions, I noticed that none of my projects can be found. So I tried manually sending request to the GitHub Rest-API and the response was always `[ ]`. However, using GraphQL, I could access a list of my org's projects.

So my question is: **Can't I use REST-API with modern projects?**

**Comment by mvkaran** [comment, marked as answer]
Hi @marcoschwepp 👋🏼 

As of now, the new Projects is available only on the GraphQL API.

Curious to know if you have any specific use case where you'd want to use only REST and not GraphQL for the new Projects.

**Reply by marcoschwepp** [reply]
Hey @mvkaran,
thanks for your awnser! My problem was that i wanted to use existing actions, but those were using REST. 
However in the meantime i have found an action which is using GraphQL, so my problem is solved.

Thanks for your time!
